src/index.js

```diff
@@ -1,7 +1,7 @@
 'use strict';
 
 var assign = require('lodash.assign');
-var ResultList = require('./result-list');
+let ResultList = require('./result-list');
```

Comment:
'let' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).